src/Functions/buildId.cpp

```diff
@@ -42,6 +47,9 @@ class FunctionBuildId : public IFunction
     {
         return DataTypeString().createColumnConst(input_rows_count, SymbolIndex::instance()->getBuildIDHex());
     }
+
+private:
+    ContextPtr context;
```

Comment:
Let's better store only `is_distributed` flag instead of whole context.

Reply:
Done